hellosign-embedded/index.d.ts

```diff
@@ -268,3 +268,5 @@ declare module HelloSign {
 }
 
 declare var HelloSign: HelloSign.HelloSignStatic;
+
+export = HelloSign;
```

Comment:
If I try to do that, I get a TypeScript error: "Global module exports may only appear in module files."